app/javascript/mastodon/features/ui/components/image_loader.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ImmutablePureComponent from 'react-immutable-pure-component';
+import PropTypes from 'prop-types';
+
+class ImageLoader extends ImmutablePureComponent {
```

Comment:
ImmutablePureComponent is for Immutable.js. Immutable.js is not used in this component, so I think it is unnecessary.